packages/financial-templates-lib/src/price-feed/CreatePriceFeed.js

```diff
@@ -162,6 +163,31 @@ async function createPriceFeed(logger, web3, networker, getTime, config) {
       config.denominatorPriceFeed && (await _createMedianizerPriceFeed(config.denominatorPriceFeed));
 
     return new BasketSpreadPriceFeed(web3, logger, baselinePriceFeeds, experimentalPriceFeeds, denominatorPriceFeed);
+  } else if (config.type === "tradermade") {
+    const requiredFields = ["pair", "apiKey", "botLookback", "affiliateLookback", "minTimeBetweenUpdates"];
+
+    if (isMissingField(config, requiredFields, logger)) {
+      return null;
+    }
+
+    logger.debug({
+      at: "createPriceFeed",
+      message: "Creating TraderMadePriceFeed",
+      config
+    });
+
+    return new TraderMadePriceFeed(
+      logger,
+      web3,
+      config.apiKey,
+      config.pair,
+      config.botLookback,
+      config.affiliateLookback,
```

Comment:
I like how you broke out the 2 lookback params, but it breaks the expected API, at least for affiliates, as it expects only a `lookback` param.  I think though just leave this for now and I will change it on my end.

Reply:
just a  nit here but maybe it might be clearer to understand what these are doing to rename them to something like:
`botLookback -> minuteLookback`
`affiliateLookback -> hourlyLookback`

Reply:
changed